src/ButtonInput.js

```diff
@@ -17,14 +17,14 @@ class ButtonInput extends InputBase {
   }
 }
 
-ButtonInput.types = ['button', 'reset', 'submit'];
+ButtonInput.types = Button.types;
```

Comment:
Fair enough. I don't know that anybody should be depending on this, but I guess you're being more careful than me (:

Could you modify the `propTypes` validation below to use this `ButtonInput.types` then, just in case someone replaces `ButtonInput.types`?